client-v2/src/shared/README.md

```diff
@@ -12,36 +12,23 @@ The shared collections library provides:
 The consumer application is responsible for fetching data (collections, details of the articles in the collections) from its own data store and from an external store in (e.g. capi) and passing
 this data on to the shared collections module.
 
+Collections and articles should be passed to the shared collections library in a normalised. There are utility functions to normalised collections as they are stored in facia-tool or story packages but if the consumer uses a different datastructure, they will have to do perform this normalisation. The `Data types in the library` section contains details of what normalised collections and articles look like.
+
 We assume articles are stored in the consumer application and in an external datastore because the consumer application should never edit the articles themselves as they appear in capi. It only edits the way articles are displayed in a particular collection. To understand what an article looks like, we look at how that article appears in the external store and what edits we have applied to that article within our application.
 
 The consumer application also controls the stage of the collection we want to view, e.g. in the case of the fronts tool whether we want to render draft or live collections.
 
-## Using the library
-
-### Fetching data
-
-To use the library, you have to dispatch three actions: `articleFragmentsReceived`, `externalArticlesReceived` and `collectionReceived` actions.
-
-### External Articles
-External articles represent the articles as they exist in external to your application e.g. in the content api.
-
-They look like this:
+## Data types in the library
 
-```
-type ExternalArticle = {
-  id: string,
-  headline: string
-};
-```
+There are few data types in this library which represent articles and collections.
```

Comment:
I guess this is 'there are a few', which is a bit pedantic but could potentially be misleading!